### DISCUSSION TITLE
Extend the focus mode with scheduling

### DISCUSSION THREAD

**Discussion by SansGuidon** [dailydotdev/daily#496] [Feature Requests]
I've configured daily.dev to open in every new tab, which is impeding my focus. I would like to be able to configure daily.dev to only display its content at certain times on certain days.

**Comment by idoshamun** [comment]
Hi @MorganGeek,

That's a great idea. Let me raise it in our upcoming product planning meeting.
Thanks a lot :)

**Reply by SansGuidon** [reply]
Thank you very much @idoshamun  :)
